climatepolicyradar/navigator-search-indexer
# Separate search indexer from main navigator repo

contains (from [linear](https://linear.app/climate-policy-radar/issue/BAK-1112/separate-indexer-and-pdf-parser-into-their-own-docker-images)):
* move across methods from common folder
* set up with docker & tests
* set up pre-commit 
* set up github actions for tests (note GH actions has not run because `test.yml` hasn't been committed to the main branch, but `make build && make test` passes for me locally)
* add changelog and update readme
* check that it works with the navigator docker-compose stack

I've also set up branch protection, copying what's in the navigator repo, and set up slack notifications for this repo.
